jni/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ target_include_directories(${TARGET_LIB_COMMON} PRIVATE ${CMAKE_CURRENT_SOURCE_D
 set_target_properties(${TARGET_LIB_COMMON} PROPERTIES SUFFIX ${LIB_EXT})
 set_target_properties(${TARGET_LIB_COMMON} PROPERTIES POSITION_INDEPENDENT_CODE ON)
 
-if (WIN32)
+if (NOT "${WIN32}" STREQUAL "")
```

Comment:
is ```if (WIN32)``` and ```if (NOT "${WIN32}" STREQUAL "")``` are same?

Reply:
WIN32 is not a variable it is just the string "WIN32". "${WIN32}" is a variable.